fastlane/Fastfile

```
@@ -0,0 +1,146 @@
+    lane :build_test_r do |options|
+        keystore_path = File.expand_path('../keystore.jks')
+        gradle(
+                  task: "clean bundle#{options[:variant]}Release",
+
+              )
+    end
+
+  desc "Build and Distribute Debug APK to Firebase"
+  lane :build_and_distribute_debug do |options|
+
+   latest_release = firebase_app_distribution_get_latest_release(
```

RuboCop (1.69.1)

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)